### DISCUSSION TITLE
Need help to build and run android code

### DISCUSSION THREAD

**Discussion by chandralabs** [NightscoutFoundation/xDrip#3197] [Q&A]
@jamorham, @Navid200 and @jtrebicki, I am an independent researcher and a diabetic patient, want to use nights out with my backend using xDrip+. I am facing issues while build the recent Nightly build 16th Jul 2023 and also master branch,
Could you please share the steps to build and run the code using android studio? 
Any help, suggestions will be appreciated.

Thanks.

**Comment by Navid200** [comment]
May I ask what you are going to change?

I mean xDrip is open source.  That means everyone is welcome to contribute.  
But, if you only intend to make changes for your own benefit and have no interest in helping us improve xDrip for everyone, that's not really the idea.
